test_stack.py

```diff
@@ -0,0 +1,46 @@
+from __future__ import unicode_literals
+import pytest
```

Comment:
if you don't use any specifics of the `pytest` module, you need not import it.  here you aren't using it at all.